src/sections/wrapper.tsx

```diff
@@ -0,0 +1,282 @@
+      set_wxhoprValue('');
+    }
+  }, [is_xHOPR_to_wxHOPR_success, is_wxHOPR_to_xHOPR_success]);
+
+  // Set the maximum value for xHOPR on input field.
+  const setMax_xHOPR = () => {
+    if (xHOPR_balance) {
+      set_xhoprValue(xHOPR_balance.formatted);
+    }
+  };
+
+  const setMax_wxHOPR = () => {
+    if (wxHOPR_balance) {
+      set_wxhoprValue(wxHOPR_balance.formatted);
+    }
+  };
+
+  return (
+    <Section
+      lightBlue
+      center
+    >
+      <StyledPaper>
+        <h2>Wrapper</h2>
+        <p>Utility to wrap (xHOPR &#8594; wxHOPR) and unwrap (wxHOPR &#8594; xHOPR) xHOPR tokens.</p>
+        <WrapperContainer>
+          <StyledTextField
+            label="xHOPR"
+            placeholder="Your xHOPR here..."
+            type="number"
+            value={xhoprValue}
+            onChange={(e) => set_xhoprValue(e.target.value)}
+            disabled={swapDirection === 'wxHOPR_to_xHOPR'}
+            InputProps={{
+              endAdornment: (
+                <InputAdornment position="end">
+                  <MaxButton
+                    disabled={!write_xHOPR_to_wxHOPR || swapDirection === 'wxHOPR_to_xHOPR'}
+                    onClick={setMax_xHOPR}
+                  >
+                    Max
+                  </MaxButton>
+                </InputAdornment>
+              ),
+              inputProps: { min: 0 },
+            }}
+          />
+          <StyledIconButton
+            onClick={handleSwap}
+            disabled={!write_xHOPR_to_wxHOPR || !write_wxHOPR_to_xHOPR}
+          >
+            <SwapVertIcon />
+          </StyledIconButton>
+          <StyledTextField
+            label="wxHOPR"
+            placeholder="Your wxHOPR here..."
+            type="number"
+            value={wxhoprValue}
+            onChange={(e) => set_wxhoprValue(e.target.value)}
+            disabled={swapDirection === 'xHOPR_to_wxHOPR'}
+            InputProps={{
+              endAdornment: (
+                <InputAdornment position="end">
+                  <MaxButton
+                    disabled={!write_wxHOPR_to_xHOPR || swapDirection === 'xHOPR_to_wxHOPR'}
+                    onClick={setMax_wxHOPR}
+                  >
+                    Max
+                  </MaxButton>
+                </InputAdornment>
+              ),
+              inputProps: { min: 0 },
+            }}
+          />
+          <Button
+            hopr
+            className="swap-button"
+            disabled={!write_xHOPR_to_wxHOPR || !write_wxHOPR_to_xHOPR}
+            onClick={handleClick}
+          >
+            Swap
+          </Button>
+          {(is_xHOPR_to_wxHOPR_loading || is_wxHOPR_to_xHOPR_loading) && <span>Check your Wallet...</span>}
+          <TransactionLink
+            isSuccess={is_xHOPR_to_wxHOPR_success}
+            hash={xHOPR_to_wxHOPR_data?.hash}
+          />
+          <TransactionLink
+            isSuccess={is_wxHOPR_to_xHOPR_success}
+            hash={wxHOPR_to_xHOPR_data?.hash}
+          />
```

Comment:
add more detail to be able to know what tx wrap it refers to

Reply:
Done